inst/rmarkdown/templates/mortality/skeleton/skeleton.Rmd

```diff
@@ -1,120 +1,236 @@
+  kable(digits = 2)
+```
+
+Age distribution of current household population by five year age groups and by gender.
+```{r age_pyramid}
+plot_age_pyramid(study_data_cleaned, 
+                 age_group = "age_group", 
+                 split_by = "sex") + 
+  labs(x = "Cases (n)", y = "Age group (years)") +                    # change axis labels
+  theme(legend.position = "bottom", legend.title = element_blank())   # remove title and move legend
+```
+
+```{r}
+# TODO: Table - Age distribution of the sample population and of the study population.
+```
 
 
 ## Mortality
 
+Reported causes of death and cause-specific mortality rates (cause-specific mortality per 10000 still missing!)
+```{r}
+death_tbl <- function(x) {
+  x %>% 
+    filter(!is.na(cause_of_death),
+           died) %>%  # remove rows with missing outcome
+    group_by(cause_of_death) %>% 
+    summarise(deaths = n()) %>%  # deaths by cause of death
+    # calculate case fatality rate
+    do(bind_cols("cause_of_death" = .$cause_of_death, 
+                 case_fatality_rate(.$deaths, sum(.$deaths), mergeCI = TRUE))) %>%
+    tidyr::complete(cause_of_death, fill = list(deaths = 0)) %>%                 # Ensure all levels are represented
+    mutate(cfr = if_else(deaths == 0, "-", paste0(round(cfr, 1), " ", ci))) %>% 
+    select(cause_of_death, deaths, cfr) %>% 
+    rename("Reported cause of death" = cause_of_death,
+           "n" = deaths,
+           "% (95% CI)" = cfr)
+}
+# todo: make cause of death factor!
+death_tbl(study_data_cleaned %>% 
+  filter(!is.na(cause_of_death),
+         died)) %>%
+  knitr::kable(digits = 2) 
+
+```
 
+Cause of deaht by age group
 ```{r}
-suvery_ciprob(suvery_design, "died") %>%
-  knitr::kable()
+
+d1 <- death_tbl(study_data_cleaned %>% 
+  filter(!is.na(cause_of_death),
+         died,
+         age_in_years %in% 0:4)) %>% 
+  rename("n <5 years old" = n)
+d2 <- death_tbl(study_data_cleaned %>% 
+  filter(!is.na(cause_of_death),
+         died,
+         age_in_years %in% 5:14)) %>% 
+  rename("n 5 to 14 years old" = n)
+d3 <- death_tbl(study_data_cleaned %>% 
+  filter(!is.na(cause_of_death),
+         died,
+         age_in_years %in% 15:60)) %>% 
+  rename("n 15 to 60 years old" = n)
+d4 <- death_tbl(study_data_cleaned %>% 
+  filter(!is.na(cause_of_death),
+         died,
+         age_in_years >60)) %>% 
+  rename("n 60+ years old" = n)
+
+cbind(d1, d2[, 2:3], d3[, 2:3], d4[, 2:3]) %>%
+  knitr::kable(digits = 2) 
+
+
 ```
 
+Mortality rates and reported causes of death by gender
 ```{r}
-suvery_ciprob_stratified(suvery_design, "died", "age_group") %>%
-  kable()
+d1 <- death_tbl(study_data_cleaned %>% 
+  filter(!is.na(cause_of_death),
+         died,
+         sex == "Female")) %>% 
+  rename("n Female" = n)
+d2 <- death_tbl(study_data_cleaned %>% 
+  filter(!is.na(cause_of_death),
+         died,
+         sex == "Male")) %>% 
+  rename("n Male" = n)
+
+cbind(d1, d2[, 2:3]) %>%
+  knitr::kable(digits = 2) 
+
 ```
 
+## Morbidity
```

Comment:
alex to check with the example document - if we should add fake data to the dataset....

Reply:
@epiamsterdam can we dsicuss this quick